data/json/items/comestibles/sandwich.json

```diff
@@ -110,6 +110,24 @@
     "fun": 15,
     "vitamins": [ [ "vitC", 6 ], [ "calcium", 7 ], [ "iron", 12 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "sandwich_fairy",
```

Comment:
this needs a calorie value

Reply:
Done. Not quite familiar with this inheritance stuff yet, threw myself off there.